includes/wikia/api/DWDimensionApiControllerSQL.class.php

```diff
@@ -42,6 +42,15 @@ class DWDimensionApiControllerSQL {
             page_wikia_id, page_id
         LIMIT $limit';
 
+	const DIMENSION_WIKI_ARTICLES = '
+		SELECT
+            page_namespace AS namespace_id,
+            page_id AS article_id,
+            page_title AS title,
+            page_is_redirect AS is_redirect,
+            (SELECT MIN(rev_timestamp) from revision r where r.rev_page = p.page_id) as created_at
```

Comment:
I believe a join instead of subquery can work here since joining on `rev_page = page_id` should return the oldest revision (needs verification though)